docs/source/essentials/30-custom-scalars.mdx

```diff
@@ -81,29 +88,36 @@ If you prefer working with `Maps`, Apollo Android comes with `AnyAdapter` that s
 }
 ```
 
-This solution is more concise albeit slightly less performant.
+This solution is more concise but slightly less performant.
 
-## Registering your adapter
+## Register adapters
 
-Use `ApolloClient.withCustomScalarAdapter` to register a custom scalar Adapter. It takes a typesafe generated class from `Types` as input. ()If you can't find `Types`, build your project to trigger codegen.)
+After you define your adapters, you need to register them with your `ApolloClient` instance. To do so, call `ApolloClient.withCustomScalarAdapter` once for each adapter:
 
 ```kotlin
 val apolloClient = ApolloClient("https://").withCustomScalarAdapter(Types.GeoPoint, geoPointAdapter)
 ```
 
-## Apollo Adapters
+This method takes a type-safe generated class from `Types`, along with its corresponding adapter.
+
+> If you can't find `Types`, build your project to trigger codegen.
 
-Apollo comes with builtin adapters for common scalar types in the `com.apollographql.apollo3:apollo-adapters` artifact. It provides:
 
-* com.apollographql.apollo3.adapter.InstantAdapter for `kotlinx.datetime.Instant` ISO8601 dates
-* com.apollographql.apollo3.adapter.LocalDateAdapter for `kotlinx.datetime.LocalDate` ISO8601 dates
-* com.apollographql.apollo3.adapter.DateAdapter for `java.util.Date` ISO8601 dates
-* com.apollographql.apollo3.adapter.LongAdapter for `java.lang.Long`
-* com.apollographql.apollo3.adapter.BigDecimalAdapter for a MPP `com.apollographql.apollo3.BigDecimal` class holding big decimal values
+## Apollo-provided adapters
 
-For an example, to use `DateAdapter`, configure your Gradle scripts:
+The `com.apollographql.apollo3:apollo-adapters` artifact provides built-in adapters for the following common custom scalar types.
 
-```koltin
+| Adapter | Description |
+|---------|-------------|
+| `com.apollographql.apollo3.adapter.InstantAdapter` | For `kotlinx.datetime.Instant` ISO8601 dates |
+| `com.apollographql.apollo3.adapter.LocalDateAdapter` | For `kotlinx.datetime.LocalDate` ISO8601 dates |
+| `com.apollographql.apollo3.adapter.DateAdapter` | For `java.util.Date` ISO8601 dates |
+| `com.apollographql.apollo3.adapter.LongAdapter` | For `java.lang.Long` |
+| `com.apollographql.apollo3.adapter.BigDecimalAdapter` | For a MPP `com.apollographql.apollo3.BigDecimal` class holding big decimal values |
```

Comment:
Nice , way more readable 👍 

I'll copy this over to https://github.com/apollographql/apollo-android/blob/dev-3.x/apollo-adapters/README.md